src/main/kotlin/app/revanced/patches/youtube/layout/theme/bytecode/patch/ThemeBytecodePatch.kt

```diff
@@ -78,7 +80,18 @@ class ThemeBytecodePatch : BytecodePatch(
         return PatchResultSuccess()
     }
 
-    private companion object {
+    companion object {
```

Comment:
Why has the visibility been changed? If you intend the method to be used by other patches, it should be added in a utility class for all patches.